app/controllers/users_controller.rb

```diff
@@ -0,0 +1,39 @@
+class UsersController < ApplicationController
+  def create
+    @user = User.new(user_params)
+
+    if @user.save
+      puts "user saved"
```

Comment:
You may use the rails `flash[:success]` hash instead. [OPTIONAL]

Reply:
Also, this does nothing to give feedback to users but only the cli. Truly consider using the `flash` hash